pointing_frame: $(arg prefix)outside_camera_color_optical_frame
min_duration: 0.05
max_velocity: 0.7